en/components/grids_templates/cell-editing.md

```diff
@@ -181,6 +181,85 @@ This code is used in the sample below which implements an [`IgxSelectComponent`]
 
 For more information on how to configure columns and their templates, you can see the documentation for [Grid Columns configuration](../grid/grid.md#angular-grid-column-configuration).
 
+@@if (igxName === 'IgxGrid') {
+
+### @@igComponent Excel Style Editing
+
+
+Using Excel Style Editing allows the user to navigate trough the cells just as he would using the Excel, and ever so quickly edit them.
+
+Implementing this custom functionality can be done by utilizing the events of the grid. First we hook up to the grid's keydown events, and from there we can implement two functionalities:
+
+* Constant edit mode
+
+
+```typescript
+    public keydownHandler(event) {
+      const key = event.keyCode;
+      const grid = this.grid;
+      const activeElem = grid.navigation.activeNode;
+
+      if (
+        (key >= 48 && key <= 57) ||
+        (key >= 65 && key <= 90) ||
+        (key >= 97 && key <= 122)
+      ) {
+        // Number or Alphabet upper case or Alphabet lower case
+
+        const columnName = grid.getColumnByVisibleIndex(activeElem.column).field;
+        const cell = grid.getCellByColumn(activeElem.row, columnName);
+        if (cell && !grid.crudService.cellInEditMode) {
+          grid.crudService.enterEditMode(cell);
+          cell.editValue = event.key;
+        }
+      }
+    }
+```
+
+* `Enter`/ `Shift+Enter` navigation
+
+```typescript
+
+// should we hit enter, we use the grid's navigation tools to get to the bottom cell instead
+if (key == 13) {
+let thisRow = activeElem.row;
+const column = activeElem.column;
+const rowInfo = grid.dataView;
+this.grid.navigateTo(nextRow, column, (obj) => {
```

Comment:
This snipped for `enter` key is missing some closing brackets and a line of code before the `if (key == 13)`
This is the line `let nextRow = this.getNextEditableRowIndex(thisRow, rowInfo, event.shiftKey);`

Reply:
should be looking good now